apis_configs/fence_creds.json

```diff
@@ -0,0 +1,10 @@
+{
+  "db_host": "postgres",
+  "db_username": "fence_user",
+  "db_password": "fence_pass",
+  "db_database": "fence_db",
+  "hostname": "localhost",
```

Comment:
Lets add a note about updating this too if you are running it on a remote machine (i.e. EC2 instance)

Reply:
Is this referring to trying to run the Docker Compose setup in production on a remote host? I haven't tried this, I think there might be more things to tweak, such as adding a Swarm setup.

Reply:
So, this would essentially just be changing the hostname?

Reply:
Yep, we just need a note in the readme on how to set this up and where to change it